server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/VacationIntegrationTest.java

```diff
@@ -168,7 +170,6 @@ public void jmapVacationShouldNotGenerateAReplyWhenInactive() throws Exception {
         */
 
         // Given
-        guiceJamesServer.serverProbe().createMailbox(MailboxConstants.USER_NAMESPACE, USER_2, "INBOX");
```

Comment:
No.

Await waits for indexing event to be ended in ElasticSearch.

Mailbox added envents do not modify anything in Elastic search

createMailbox call is moreover synchronous.

The error I correct here was not for this method, but for the over one, that actually sends a notification, and lacked this mailbox creation logic. The code worked fast enough for this timing issue to be unspotted but adding a Cassandra query broke it...